lib/modem_jwt/modem_jwt.c

```diff
@@ -18,13 +18,6 @@
 #define JWT_CMD_PRINT_INT_SZ (MAX_INT_PRINT_SZ * 4)
 #define GET_BASE64_DECODED_LEN(n) (3 * n / 4)
 
-/* Minimum JWT response is ~420 bytes.
```

Comment:
the information in this comment might be useful in the help text for `MODEM_JWT_MAX_LEN`

Reply:
Added to help text